pytests/tests/test_check_update.py

```diff
@@ -40,6 +54,15 @@ def test_check_update_multi_version_package(utils):
     assert len(ret['stdout']) > 0
```

Comment:
`assert` here for the expected return code.

Reply:
done